doc/man/8/ceph-disk.rst

```diff
@@ -176,6 +193,62 @@ Usage::
 
 Here, [PATH] is path to a block device or a directory.
 
+deactivate
+----------
+Deactivate the Ceph OSD. It stops osd daemo and try to mark this osd out
+(optional). To prevent OSD activate by ``udev``, remove the *ready*, *active*,
+*INIT-specific* files and add the *deactive* file. Last, unmount the currect
+osd location. Futhermore, if the osd is dmcrypt, remove the data dmcrypt map. Atfer
+that, osd will remain the ``down`` status, you can activate with
+:option:`--reactivate`. Now, It is only run directly. It can be triggered
+by :program:`ceph-deploy` in later version.
+
+Usage::
+
+	ceph-disk deactivate [PATH]
+
+Here, [PATH] is path to a block device or a directory.
+
+Another option :option:`--mark-out` can also be used with this subcommand.
+``--mark-out`` marks the OSD out. It will trigger the object remapped. If
+you are not sure you will destory OSD, you may avoid to use this option.
+That may cause redundant overhead due to object remapped.
+
+For the convince, you can also use ``osd-id``, id of osd, to deactivate
+osd with subcommand :option:`--deactivate-by-id`.
+
+Usage::
+
+	ceph-disk deactivate --deactivate-by-id [OSD-ID]
+
+destroy
+-------
+Destroy the Ceph OSD. It removes the osd from cluster and crushmap and
+deallocates OSD ID. It only can destroy the OSD on *down* status. Futhermore,
+if the osd is dmcrypt, remove the journal dmcrypt map. Now, It is only run
+directly. It could be triggered by :program:`ceph-deploy` in later version.
+
+Usage::
+
+	ceph-disk destroy [PATH]
+
+Here, [PATH] is path to a block device or a directory.
+
+Another option :option:`--zap` can also be used with this subcommand.
+``--zap`` will erase data and partition.
+
+Usage::
+
+	ceph-disk destroy [PATH] [--zap]
+
+For the convince, you can alose use ``osd-id``, id of osd, to destroy
+osd with subcommand :option:`--destroy-by-id`. It costs redundant
+overhead due to dmcrypt map rebuilding and osd-id checking.
```

Comment:
I would remove

    It costs redundant overhead due to dmcrypt map rebuilding and osd-id checking.

because it is internal and the difference in performance is unlikely to be noticed.